packages/server/src/models/user.entity.ts

```diff
@@ -13,4 +13,10 @@ export default class User {
 
   @Column()
   public password: string;
+
+  @Column()
+  public facebook_id: string;
```

Comment:
Consider naming fields in camel case.

Reply:
I do not mind. Just thought that in DBs they usually name fields this way

Reply:
I think it's better for code consistency